cypress/page/products-list-page.ts

```diff
@@ -1,26 +1,11 @@
 class ProductsListPage {
   private productList: string;
-  private proceedToCheckoutButton: string;
-  private productos: string;
 
   constructor() {
-    this.productList = "a[title='Add to cart']";
-    this.proceedToCheckoutButton =
-      ".button-container a[title='Proceed to checkout']";
-    this.productos = ".product-container";
+    this.productList = "#item_1_title_link > .inventory_item_name";
```

Comment:
Is there a way to avoid querying by class?

Reply:
It will be changed in the next steps